xonsh/environ.py

```diff
@@ -14,6 +14,7 @@
 import subprocess
 import platform
 
+from pathlib import Path
```

Comment:
This should just be `import pathlib` and grouped with the standard library imports

Reply:
Fixed